native/src/main/java/org/ballerinax/asb/admin/Administrator.java

```diff
@@ -0,0 +1,999 @@
+        return createRecordValue(ModuleUtils.getModule(), ASBConstants.LIST_OF_QUEUES_RECORD, map);
+    }
+
+    /**
+     * Delete a Queue in Azure Service Bus.
+     *
+     * @param administratorClient Azure Service Bus Administrator Client
+     * @param queueName           name of the Queue
+     * @return null
+     */
+    public static Object deleteQueue(BObject administratorClient, BString queueName) {
+        ServiceBusAdministrationClient clientEp = getAdminFromBObject(administratorClient);
+        try {
+            clientEp.deleteQueue(queueName.toString());
+            LOGGER.debug("Deleted queue successfully with name: " + queueName);
+            return null;
+        } catch (BError e) {
+            return ASBErrorCreator.fromBError(e);
+        } catch (ServiceBusException e) {
+            return ASBErrorCreator.fromASBException(e);
+        } catch (Exception e) {
+            return ASBErrorCreator.fromUnhandledException(e);
+        }
+    }
+
+    /**
+     * Check whether a Queue exists in Azure Service Bus.
+     *
+     * @param administratorClient Azure Service Bus Administrator Client
+     * @param queueName           name of the Queue
+     * @return null
+     */
+    public static Object queueExists(BObject administratorClient, BString queueName) {
+        ServiceBusAdministrationClient clientEp = getAdminFromBObject(administratorClient);
+        try {
+            return clientEp.getQueueExists(queueName.toString());
+        } catch (HttpResponseException e) {
+            if (e.getResponse().getStatusCode() == 404) {
+                return false;
+            }
+            return ASBErrorCreator.fromASBHttpResponseException(e);
+        } catch (BError e) {
+            return ASBErrorCreator.fromBError(e);
+        } catch (ServiceBusException e) {
+            return ASBErrorCreator.fromASBException(e);
+        } catch (Exception e) {
+            return ASBErrorCreator.fromUnhandledException(e);
+        }
+    }
+
+    private static BMap<BString, Object> constructRuleCreatedRecord(RuleProperties properties) {
+        Map<String, Object> map = populateRuleOptionalFieldsMap(properties);
+        return ValueCreator.createRecordValue(ModuleUtils.getModule(),
+                ASBConstants.CREATED_RULE_RECORD, map);
+    }
+
+    private static BMap<BString, Object> constructQueueCreatedRecord(QueueProperties properties) {
+        Map<String, Object> map = populateQueueOptionalFieldsMap(properties);
+        return ValueCreator.createRecordValue(ModuleUtils.getModule(),
+                ASBConstants.QUEUE_CREATED_RECORD, map);
+    }
+
+    private static Map<String, Object> populateTopicOptionalFieldsMap(TopicProperties properties) {
+        Map<String, Object> map = new HashMap<>();
+        ASBUtils.addFieldIfPresent(map, ASBConstants.CREATED_TOPIC_RECORD_FIELD_NAME
+                , StringUtils.fromString(properties.getName()));
```

Comment:
```suggestion
        ASBUtils.addFieldIfPresent(map, ASBConstants.CREATED_TOPIC_RECORD_FIELD_NAME, 
                StringUtils.fromString(properties.getName()));
```

Reply:
Lets follow the similar convention for all the other fields below, to have a consistent formatting